hyperstart.go

```diff
@@ -334,14 +373,57 @@ func (h *hyper) startContainer(pod Pod, contConfig ContainerConfig) error {
 		return err
 	}
 
+	err = h.exec(pod, contConfig, contConfig.Cmd)
```

Comment:
I don't know why but I thought it was not possible to execute a workload on a container if this one was not running with a pause binary. I have just made a test and I realized that it was working fine as long as I have at least one container (the pause container) running. That's why I was starting every container with the pause binary first.